### PR TITLE
XW-4331 | Update play icon for videos on Special:Videos and embedded in articles

### PR DESCRIPTION
**Pull request by rybmat**
https://wikia-inc.atlassian.net/browse/XW-4330
https://wikia-inc.atlassian.net/browse/XW-4331

updated play icon for videos:
* on special:videos
* in article (plain video, in infobox, in galleries)
* video.wikia.com

@Wikia/x-wing

### CODE REVIEW

File: extensions/VisualEditor/wikia/modules/ve/ui/widgets/ve.ui.WikiaMediaPageWidget.js [hunk 1]
```diff
@@ -217,8 +217,17 @@ ve.ui.WikiaMediaPageWidget.prototype.setupLicense = function ( $license ) {
  * @method
  */
 ve.ui.WikiaMediaPageWidget.prototype.setupVideoOverlay = function () {
-	this.$overlay = this.$( '<span>' )
-		.addClass( 'play-circle' );
+	this.$overlay = this.$( '<span>' ).addClass( 'thumbnail-play-icon-container ve-no-shield' ).html(
```
Comment (latata) [inline extensions/VisualEditor/wikia/modules/ve/ui/widgets/ve.ui.WikiaMediaPageWidget.js]: `ve-no-shield`?

File: extensions/VisualEditor/wikia/modules/ve/tests/ce/ve.ce.wikiaExample.js [hunk 2]
```diff
@@ -104,7 +104,17 @@ ve.ce.wikiaExample = ( function ( utils ) {
 
 	media.html.video = {
 		playButton:
-			'<span class="play-circle ve-no-shield"></span>'
+		'<span class="thumbnail-play-icon-container">' +
```
Comment (latata) [inline extensions/VisualEditor/wikia/modules/ve/tests/ce/ve.ce.wikiaExample.js:107]: There should be `ve-no-shield` I think...

File: extensions/VisualEditor/wikia/modules/ve/ce/ve.ce.WikiaVideoNode.js [hunk 3]
```diff
@@ -72,7 +72,17 @@ ve.ce.WikiaVideoNode.static.getSizeString = function ( width ) {
  * @returns {jQuery}
  */
 ve.ce.WikiaVideoNode.prototype.createPlayButton = function () {
-	return this.$( '<span>' ).addClass( 'play-circle ve-no-shield' );
+	return this.$( '<span>' ).addClass( 'thumbnail-play-icon-container ve-no-shield' ).html(
+		'<svg class="thumbnail-play-icon" viewBox="0 0 180 180" width="100%" height="100%">' +
```
Comment (rybmat) [inline extensions/VisualEditor/wikia/modules/ve/ce/ve.ce.WikiaVideoNode.js]: @hakubo we should take care about https://wikia-inc.atlassian.net/browse/XW-3926 sooner than later to not have to do such ugly things like pasting svg markup into js files

File: extensions/wikia/VideoPageTool/templates/VideoHomePageController_featured.php [hunk 4]
```diff
@@ -5,8 +5,10 @@
 			<? $index = 0; ?>
 			<? foreach ( $assets as $videoData ): ?>
 				<li>
-					<div class="slide-image video video-thumbnail xlarge hide-play fluid" data-index="<?= $index ?>">
-						<span class="play-circle"></span>
+					<div class="slide-image video video-thumbnail xlarge fluid" data-index="<?= $index ?>">
+						'<span class="thumbnail-play-icon-container">'
```
Comment (latata) [inline extensions/wikia/VideoPageTool/templates/VideoHomePageController_featured.php]: missing `<?`/`?>`?

File: extensions/VisualEditor/wikia/modules/ve/tests/ce/ve.ce.wikiaExample.js (hunk 2 above)
Comment (rybmat) [reply extensions/VisualEditor/wikia/modules/ve/tests/ce/ve.ce.wikiaExample.js:107]: this class was not used in css at all

File: extensions/wikia/VideoPageTool/templates/VideoHomePageController_featured.php (hunk 4 above)
Comment (rybmat) [reply extensions/wikia/VideoPageTool/templates/VideoHomePageController_featured.php]: done, line below

File: extensions/wikia/Thumbnails/templates/Thumbnail_video.mustache [hunk 5]
```diff
@@ -8,7 +8,7 @@
 	{{>Thumbnail_imgTag}}
 	{{#noscript}}<noscript>{{{noscript}}}</noscript>{{/noscript}}
 
-	{{#duration}}<span class="duration" itemprop="duration">{{duration}}</span>{{/duration}}
```
Comment (latata) [inline extensions/wikia/Thumbnails/templates/Thumbnail_video.mustache:11]: That will remove duration from video metadata. Is it ok?

**Review by latata — APPROVED** [review]

File: extensions/wikia/Thumbnails/templates/Thumbnail_video.mustache (hunk 5 above)
Comment (rybmat) [reply extensions/wikia/Thumbnails/templates/Thumbnail_video.mustache:11]: currently https://search.google.com/structured-data/testing-tool#url=http%3A%2F%2Frybatest2.wikia.com%2Fwiki%2FTesting_video does not detect any of included videos, so I think it is ok. This tag didn't conform to schema.org spec https://developers.google.com/webmasters/videosearch/schema

Comment (rybmat) [reply extensions/wikia/Thumbnails/templates/Thumbnail_video.mustache:11]: We could add such meta tags to every video, but I think we should consult it with stanley

**Review by hakubo — APPROVED** [review]

**Pull request merged**